subsys/usb/class/hid/Kconfig

```diff
@@ -15,6 +15,12 @@ config USB_DEVICE_HID
 
 if USB_DEVICE_HID
 
+config ENABLE_HID_INT_OUT_EP
+	bool
+	prompt "Enable USB HID Device Interrupt Endpoint"
+	help
+	  Enable USB HID Device Interrupt Endpoint.
+
```

Comment:
I think the help and the prompt, should say "Interrupt OUT Endpoint" to avoid confusion.